packages/graphile-build-pg/res/introspection-query.sql

```diff
@@ -102,7 +102,7 @@ with
       (pg_catalog.pg_relation_is_updatable(rel.oid, true)::bit(8) & B'00000100') = B'00000100' as "isDeletable"
     from
       pg_catalog.pg_class as rel
-      left join pg_catalog.pg_description as dsc on dsc.objoid = rel.oid and dsc.objsubid = 0
+      left join pg_catalog.pg_description as dsc on dsc.objoid = rel.oid and dsc.objsubid = 0 and dsc.classoid = (select oid from pg_catalog.pg_class where relname = 'pg_class')
```

Comment:
Unfortunately this discards comments on things that aren't tables/columns, such as functions.

```
[store] # select oid, relname from pg_class where oid in (select distinct classoid from pg_description);
 oid  |    relname
------+----------------
 1247 | pg_type
 1255 | pg_proc
 1259 | pg_class
 2617 | pg_operator
 2601 | pg_am
 2612 | pg_language
 2615 | pg_namespace
 2607 | pg_conversion
 3602 | pg_ts_config
 3600 | pg_ts_dict
 3601 | pg_ts_parser
 3764 | pg_ts_template
 3079 | pg_extension
 3456 | pg_collation
(14 rows)